packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -146,6 +146,12 @@ abstract class FlutterCommand extends Command<void> {
         'will select a random open port on the host.',
       hide: hide,
     );
+    argParser.addFlag('web-skip-launch',
```

Comment:
To avoid double-negatives (`--no-web-skip-launch`), maybe use a positive flag that defaults to `true`? `launch-browser`, then use `--no-launch-browser` when you want it off?

(also applies to the `skipWebLaunch` field, although it's not developer-facing, so I'm less concerned)

Reply:
Updated